Assignment4/alexfcab-200385/task06.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1h0fyE_7g4Y9jYROMCdizes5Qa-6RbvHK
+
+**Task 06: Modifying RDF(s)**
+"""
+
+# !pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**
+
+"""
+
+# Visualize the results
+g.add((ns.University, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+# Visualize the results
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+# Visualize the results
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+
+# Visualize the results
+vcard = Namespace("https://www.w3.org/2001/bcard-rdf/3.0#")
+g.add((ns.JaneSmith, vcard.FN, Literal("Jane Smith")))
+g.add((ns.Jane, vcard.Given, Literal("Jane")))
+g.add((ns.Smith, vcard.Family, Literal("Smith")))
+for s,p,o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+g.add((ns.UPM, RDF.type, ns.University))
+
+g.add((ns.worksAt, RDFS.domain, ns.Person))
+g.add((ns.worksAt, RDF.type, ns.Property))
+g.add((ns.worksAt, RDFS.range, ns.University))
+
+g.add((ns.JohnSmith, vcard.worksAt, ns.UPM))
```

Comment:
worksAt does not exist neither in vcard. Create your own under the ns  namespace